api/src/api/notes.js

```diff
@@ -12,29 +13,57 @@ const {
   validateReqParams,
 } = require('../middleware/notes');
 
+const replaceIdWithMember = async (ids) => {
```

Comment:
'replace' implies a side effect. How about we call this something like `memberFromId`